### PR TITLE
Tac stats

### PR DESCRIPTION
**Pull request by gzagatti**
### I have:

- [x] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [x] Added tests for any new additions
- [x] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [x] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

This PR adds a `HandsetStats` query class to calculate statistics out of numeric variables in the TAC table.

### CODE REVIEW

File: flowmachine/tests/test_subscriber_handset_stats.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# -*- coding: utf-8 -*-
+
+from flowmachine.features.subscriber.handset_stats import *
+
+import pytest
+
+from flowmachine.features import SubscriberHandsets
+
+# the test db contains no handset information so we can only test if the query
```
Comment (greenape) [inline flowmachine/tests/test_subscriber_handset_stats.py]: Test data does include a teeny tiny fake tac db. Augment that so this can actually be tested?

Comment (gzagatti) [reply flowmachine/tests/test_subscriber_handset_stats.py]: Fake data added.

File: flowmachine/flowmachine/features/subscriber/handset_stats.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+            raise ValueError(
+                f"{self.statistic} is not a valid statistic. Use one of {valid_stats}"
+            )
+
+        if self.characteristic not in valid_characteristics:
+            raise ValueError(
+                f"{self.characteristic} is not a valid characteristic. Use one of {valid_characteristics}"
+            )
+
+        self.handsets_query = SubscriberHandsets(
```
Comment (greenape) [inline flowmachine/flowmachine/features/subscriber/handset_stats.py]: I know I sound like a broken record, but why not take this as an argument?

Comment (gzagatti) [reply flowmachine/flowmachine/features/subscriber/handset_stats.py]: Ok, made this as argument.

File: flowmachine/flowmachine/features/subscriber/handset_stats.py [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    statistic : {'count', 'sum', 'avg', 'max', 'min', 'median', 'mode', 'stddev', 'variance'}, default 'sum'
+        Defaults to sum, aggregation statistic over the durations.
+    subscriber_handsets: flowmachine.features.subscriber_tacs.SubscriberHandsets
+        An instance of SubscriberHandsets listing the handsets associated with
+        the subscribers during the period of observation.
+
+    Examples
+    --------
+
+    >>> s = HandsetStats("2016-01-01", "2016-01-08", "width")
```
Comment (greenape) [inline flowmachine/flowmachine/features/subscriber/handset_stats.py]: Example doesn't match the signature now.

Comment (gzagatti) [reply flowmachine/flowmachine/features/subscriber/handset_stats.py]: Done.

**Pull request merged**